Header header
#The iamge width and height in pixels
uint32 imageWidth
uint32 imageHeight

#The location of the ball center in x,y (width,height)
float64 x
float64 y
#The radius of the detected ball
float64 radius
